packages/manager/apps/account-disable-2fa/src/index.tsx

```diff
@@ -1,27 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
+import { decodeToken, extractToken } from '@/utils/token';
 import initI18n from './i18n';
+import initAuthenticationInterceptor from '@/data/authentication.interceptor';
 import initInterceptor from './data/invisible-challenge.interceptor';
 
 import './global.css';
 import './index.scss';
 
-// const token = ???;
-// const user = decodeJWT(token);
-const user = {
-  legalform: 'individual',
-  email: 'monsieur.madame@ovhcloud.com',
-  language: 'fr_FR',
-  subsidiary: 'FR',
-};
-const locale = user.language;
+const token = extractToken();
+const user = decodeToken(token);
+const { language: locale, subsidiary } = user || {};
 
-initI18n(locale, [locale], user.subsidiary);
+initI18n(locale, [locale], subsidiary);
```

Comment:
can you use the initI18n from react-shell-client lib ? To avoid duplication

Reply:
Which app did you look at ?
catalog, kms, vrack-services and zimbra use initI18n from react-shell-client

Only the pci apps don't use it and duplicate it on each app

And we probably can't use a lot of things from react-shell-client lib if you don't have a container but the initI18n script is not dependant on having a container so I would rather not duplicate it one more time.

Reply:
Yes you are right, it's to retrieve the available locales